README.md

```diff
@@ -126,24 +133,20 @@ Note that if you're using Ruff alongside another LSP (like Pyright), you may wan
 LSP for certain capabilities, like `textDocument/hover`:
 
 ```lua
-local on_attach = function(client, bufnr)
-  if client.name == 'ruff_lsp' then
+
+lspconfig.ruff_lsp.setup {
```

Comment:
I think we would need to keep the `require("lspconfig")` here as the code snippets are independent of each other.